sklearn/impute/_iterative.py

```diff
@@ -474,7 +474,7 @@ def _get_abs_corr_mat(self, X_filled, tolerance=1e-6):
         abs_corr_mat = normalize(abs_corr_mat, norm='l1', axis=0, copy=False)
         return abs_corr_mat
 
-    def _initial_imputation(self, X):
+    def _initial_imputation(self, X, in_fit=True):
```

Comment:
I would find it safer to set `in_fit=False` as the default value and only set it to true when called in the fit method instead of having to explicitly set it to False anywhere else. WDYT?

Reply:
Agreed. PR updated with this change.